data-raw/pandora_column_types.tsv

```diff
@@ -96,7 +96,7 @@ TAB_Individual	individual	Id	integer	NA	NA	NA
 TAB_Individual	individual	Site	integer	NA	NA	NA
 TAB_Individual	individual	Individual_Id	integer	NA	NA	mandatory
 TAB_Individual	individual	Full_Individual_Id	character	NA	NA	NA
-TAB_Individual	individual	Owning_institution	integer	TAB_Owner	Name	mandatory
+TAB_Individual	individual	Institution_of_Origin	integer	TAB_Owner	Name	mandatory
```

Comment:
This change will only have an effect when the package data is rebuild. See `/data-raw/data_prep.R`